tsl/test/sql/continuous_aggs_usage.sql

```diff
@@ -24,7 +24,7 @@ SELECT table_name FROM create_hypertable('device_readings', 'observation_time');
 
```

Comment:
Need a test case here with the default setting for timescaledb.materialized_only with some explanation about usage and behavior.